docs/status-code-comparison-to-docker.md

```diff
@@ -0,0 +1,597 @@
+<!--[metadata]>
++++
+title = "API response codes"
+description = "Swarm API response codes"
+keywords = ["docker, swarm, response, code,  api"]
+[menu.main]
+parent="workw_swarm"
+weight=99
++++
+<![end-metadata]-->
+# Understand the Swarm vs. Engine response codes
+
+This is a documentation describing the difference between Swarm and Docker Engine in the aspect of  api response status codes.
```

Comment:
@allencloud Thanks for the contribution.  I think the description should include a bit more detail.  Not everyone coming to the page will know that they share the same API.  A bit of background is helpful.

https://gist.github.com/moxiegirl/33030590a9a32fe0c0c7

Docker Engine provides a REST API for making calls to the Engine daemon. Docker
Swarm allows a caller to make the same calls to a cluster of Engine daemons.
While the API calls are the same, the API response status codes do differ. This
document explains the differences.

Four methods are included, and they are GET, POST, PUT and DELETE.

The comparison is based on api v1.22, and all Docker Status Codes in api v1.22
are referenced from the [Engine remote API
v1.22](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/).